### PR TITLE
Updates to use a new MongoDB lib that supports php7.

### PR DESCRIPTION
**Pull request by chrisryan**
Switch the use of ext-mongo to the MongoDB PHP library. This library sits on top of the ext-mongodb for php and has a _similar_ interface to the original extension.

### CODE REVIEW

**Comment by chrisryan** [comment]
We need to update travis ci to start checking 7.

**Comment by nubs** [comment]
You can get travis ci to check against PHP 7 by adding it to the .travis.yml file

**Comment by chrisryan** [comment]
@nubs good to know, thanks. :smile:

**Comment by chrisryan** [comment]
@chadicus updated with changes for build to test 7 and use the correct driver. I don't know if this needs to wait on util or not to be updated since it does not use any of the classes that have problems.

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+  - 5.5
 env:
-  - EXT_VERSION=1.3.7 DB_PACKAGE=mongodb-10gen
-  - EXT_VERSION=1.3.7 DB_PACKAGE=mongodb-org
-  - EXT_VERSION=1.5.4 DB_PACKAGE=mongodb-10gen
-  - EXT_VERSION=1.5.4 DB_PACKAGE=mongodb-org
+  - EXT_VERSION=1.0 DB_PACKAGE=mongodb-10gen
+  - EXT_VERSION=1.0 DB_PACKAGE=mongodb-org
+  - EXT_VERSION=1.1 DB_PACKAGE=mongodb-10gen
+  - EXT_VERSION=1.1 DB_PACKAGE=mongodb-org
```
Comment (chadicus) [inline .travis.yml]: I don't think it's necessary to use version 1.0 and 1.1, i'd say just use 1.1

Comment (chrisryan) [inline .travis.yml]: removed

**Comment by chrisryan** [comment]
@chadicus ready when you are.

**Pull request merged**